devel/libevent/files/patch-apple-clang.diff

```diff
@@ -0,0 +1,32 @@
+--- util-internal.h.orig	2019-02-03 13:30:49.000000000 -0800
++++ util-internal.h	2019-07-23 22:36:20.000000000 -0700
+@@ -52,16 +52,22 @@
+ 
+ /* __has_attribute() wrapper */
+ #ifdef __has_attribute
+-#define EVUTIL_HAS_ATTRIBUTE __has_attribute
++# define EVUTIL_HAS_ATTRIBUTE __has_attribute
```

Comment:
Why the additional space? This seems unnecessary.

Reply:
You are obviously correct.  In this case, the patch comes from upstream.  Would you like me to ask upstream?

Reply:
I would not bother asking upstream. The space is perfectly valid, and actually in conditional statements can make the code easier to read, for the same reasons as c,c++ is indented. In fact, common code formatting tools like clang format often do this automatically. If this is upstreams patch, we should just go with it.

Reply:
Yah, it's weird that they did it but we should just use what upstream uses. I had incorrectly assumed it had originated with you.